src/services/bboxService.ts

```diff
@@ -39,6 +39,19 @@ export const buildBboxMap = (bboxes: IBboxLocation[], structure: AnyObject) => {
   return bboxMap;
 }
 
+export const getBboxPage = (bboxLocation: string) => {
```

Comment:
Please make sure we do not select pages for the locations that point to the bbox and includes 'pages' as part of the path, e.g. `root/document[0]/pages[0](37 0 obj PDPage)/annots[0](31 0 obj PDAnnot)`. See the file
[test_rotate.pdf](https://github.com/veraPDF/verapdf-js-viewer/files/9504993/test_rotate.pdf). You can do this in this method + rename it to show it's meaning (e.g. getSelectedPageByLocation or something like that)